ddtrace/contrib/falcon/patch.py

```diff
@@ -21,8 +22,10 @@ def patch():
 def traced_init(wrapped, instance, args, kwargs):
     mw = kwargs.pop('middleware', [])
     service = os.environ.get('DATADOG_SERVICE_NAME') or 'falcon'
+    distributed_tracing = asbool(os.environ.get(
+        'DATADOG_FALCON_DISTRIBUTED_TRACING')) or False
```

Comment:
Copied this pattern from somewhere else, but is there a smarter thing I should do?

Reply:
I'm rewriting this part because we're polluting the system with too many env vars. We can be smart here.

Reply:
Seems like we could just all use the same env var name eg DATADOG_DISTRIBUTED_TRACING. If a user runs multiple distributed services on a single host they can still just set the env vars per-executable themselves. But is a more structured config process in the pipes?

Reply:
Actually yes. For instance you may have your intake application that doesn't have distributed tracing for incoming calls (you don't accept it from clients) but you want to activate it for outgoing calls to other services. So it's still better to provide a way to select exactly what distributed tracing you want to enable.

The configuration system will make it easier, but till now I prefer that all integrations are compliant.